Mathlib/Data/Fintype/Basic.lean

```diff
@@ -755,6 +752,7 @@ theorem toFinset_univ [Fintype α] [Fintype (Set.univ : Set α)] :
   simp
 #align set.to_finset_univ Set.toFinset_univ
 
+attribute [local instance] Fintype.ofIsEmpty in
```

Comment:
Is this instance needed for the statement? Can we have a statement that merely needs `Finite` instead of `Fintype`?

Reply:
No, it's just used in the proof. Same thing for the other ones. I can try to move it inside the proofs to make this clear.